Source: Elias Pipping <pipping@exherbo.org>
Upstream: no
Reason: Be less specific about the way in which a program needs to crash
--- cmake-2.8.4/Tests/CMakeLists.txt~	2011-02-15 18:47:34.000000000 +0100
+++ cmake-2.8.4/Tests/CMakeLists.txt	2011-09-01 01:06:23.193740514 +0200
@@ -1458,7 +1458,7 @@
       PASS_REGULAR_EXPRESSION "Failed")
   ELSE(CMAKE_TEST_GENERATOR MATCHES "Watcom WMake")
     SET_TESTS_PROPERTIES(CTestTestCrash PROPERTIES
-      PASS_REGULAR_EXPRESSION "SegFault")
+      PASS_REGULAR_EXPRESSION "SegFault|Illegal")
   ENDIF(CMAKE_TEST_GENERATOR MATCHES "Watcom WMake")
 
   CONFIGURE_FILE(
